code/turf/floors_unsimulated.dm

```diff
@@ -30,6 +34,62 @@
 			return
 	..()
 
+/turf/unsimulated/floor/proc/break_tile(var/force_break)
```

Comment:
Since all these procs are basically copy pasted from `/turf/simulated/floor`, I'd rather just put them on the parent `/turf` and use vars to track whether something can be burnt/broke. 
ie add `can_burn` and `can_break` as vars on `/turf` and then move these procs there too, with
```
if (!can_break) // (or burn)
  return
```
at the top.

Reply:
(yes turf inheritance is a pain)

Reply:
yeah I thought that a base turf definition would be better, will do